test/neon-animated-pages.html

```diff
@@ -24,8 +24,8 @@
   <link rel="import" href="../../test-fixture/test-fixture.html">
   <link rel="import" href="../neon-animated-pages.html">
   <link rel="import" href="../neon-animatable.html">
-  <link rel="import" href="../animations/slide-from-top-animation.html">
-  <link rel="import" href="../animations/slide-down-animation.html">
```

Comment:
Files changed to the ones actually referenced by the animations in [used in the test](https://github.com/PolymerElements/neon-animation/blob/master/test/neon-animated-pages.html#L53).